create-publication-from-scopus/src/main/java/no/sikt/nva/scopus/xmlconverter/ScopusPublication.java

```diff
@@ -0,0 +1,207 @@
+package no.sikt.nva.scopus.xmlconverter;
```

Comment:
Is the referred task this [SMILE-1239](https://unit.atlassian.net/browse/SMILE-1239)?  I can see that the goal of the task is to create a ScopusPublication but I am wandering whether it is meaningful to convert the XML to a ScopusPublication that will then be converted to an NVA Publication. I understand that the gain of the ScopusPublication is the simplification of the Scopus XML file. 

However, given that the test coverage is low, and that the tests do not describe the expected conversion logic , e.g., tests such as `testConvert` provide little information of what the expected outcome should be. 

It could be risky to adopt this code because:
1.  we do not know how it behaves. 
2. any change in the logic will be hard to be tested since there is little test coverage. 
3. Debugging the conversion would mean that we have to look both in the conversion of the XML to ScopusPublication and from ScopusPublication to NVA-Publication. Poor test coverage makes this even more complex.
4. Any potential change in the XML-schema, can lead to breaking changes that will be hard to fix since we do not have overview of the code.

Reply:
We should have a discussion about how to approach this code. I started a good-faith review, but I have deleted the comments because they are not helpful given the fact that the code is lifted from a prototype, which is not a thing we want.